src/assets/_vars.scss

```diff
@@ -0,0 +1,41 @@
+$font-body: 'Avenir Next', sans-serif;
+$font-title: 'Moon', sans-serif;
+
+$lightBlue: #04EFFB;
+$lightBlue-o: rgba(4,239,251, .2);
+$darkBlue: #1A2A50;
+$darkBlue-o: rgba(26,42,80, .2);
```

Comment:
If you want to get real fancy, you can just do `$darkBlue-o: lighten($darkBlue,20)` (and same for lightblue, etc)
Also makes it a little more readable IMO